External/FEXCore/Source/Interface/Core/X86Tables/SecondaryTables.cpp

```diff
@@ -182,14 +182,14 @@ void InitializeSecondaryTables() {
     {0xB8, 1, X86InstInfo{"",        TYPE_INVALID, FLAGS_NONE,                                                                                      0, nullptr}},
     {0xB9, 1, X86InstInfo{"",        TYPE_GROUP_10, FLAGS_NONE,                                                                                     0, nullptr}},
     {0xBA, 1, X86InstInfo{"",        TYPE_GROUP_8, FLAGS_NONE,                                                                                      0, nullptr}},
-    {0xBB, 1, X86InstInfo{"BTC",     TYPE_INST, FLAGS_DEBUG_MEM_ACCESS | FLAGS_MODRM | FLAGS_SF_MOD_DST,                                            0, nullptr}},
-    {0xBC, 1, X86InstInfo{"BSF",     TYPE_INST, FLAGS_MODRM,                                                                                        0, nullptr}},
-    {0xBD, 1, X86InstInfo{"BSR",     TYPE_INST, FLAGS_MODRM,                                                                                        0, nullptr}},
+    {0xBB, 1, X86InstInfo{"BTC",     TYPE_INST, FLAGS_DEBUG_MEM_ACCESS | FLAGS_MODRM | FLAGS_SF_MOD_DST | FLAGS_NO_OVERLAY,                         0, nullptr}},
```

Comment:
what does FLAGS_NO_OVERLAY mean?

Reply:
FLAGS_NO_OVERLAY means that the instruction op should ignore instruction prefixes (specifically 0x66, 0xF2, and 0xF3) which could push the instruction down to another table.
This is because some of these instructions actually consume these prefixes rather than typically going to a new table.
BSF and TZCNT are good examples of this interaction.
BSF can use the 0x66 prefix (which drops it to 16bits), but the 0xF3 prefix can push it down to TZCNT in the other table (Which I have an upcoming change to make NO_OVERLAY a bit more granular)

Reply:
:+1: